tests/regression/52-apron-mukherjee/01-mukherjee_reorder_2.c

```diff
@@ -0,0 +1,70 @@
+// SKIP PARAM: --set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.activated[+] threadJoins
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <pthread.h>
+#include "assert.h"
```

Comment:
It would be more canonical to include `<assert.h>` instead. This also applies to all the following Muhkerjee benchmarks it seems.